src/slashcommands/ISlashCommand.ts

```diff
@@ -0,0 +1,6 @@
+// just guessing
+export interface ISlashCommand {
```

Comment:
I didn't get that. You mean configurable slash commands? Of course, this was only intended to be illustrative. Until then, I'd remove the slashCommands accessors from the `IConfiguration...`